dist/css/style.css

```diff
@@ -14,18 +19,68 @@ body {
     margin: 0 auto;
     padding-top: 1px;
     padding-bottom: 3.3%;
-    position: relative;
+}
+
+.header__wrapper-blackout {
+    position: fixed;
+    width: 100vw;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.01);
+    animation: hideShadow 0.25s ease;
+}
+
+.header._active .header__wrapper-blackout {
+    z-index: 10;
```

Comment:
Why 10, why not 9 or 11?

Reply:
page components: 1 - 9, shadow: 10, header menu, header button: >= 10